doc/PostgreSQL.txt

```diff
@@ -0,0 +1,129 @@
+PostgreSQL Backend
+==================
+
+The PostgreSQL backend is not for everyone. It requires some experience with
+PostgreSQL or a willness to learn. This is not something for the casual user.
+
+Status
+------
+This is an initial release, with improvements, and enhancements to come. Right
+now there is little more than the blocks table, but performance has been good.
+In time there will be mods that will make more use of the database, rather than
+flat files.
+
+Todo
+----
+- Adapt rollback to work with PostgreSQL, since rollback normally used SQLite3
+- Adapt mapgen to use PostgreSQL
```

Comment:
Ah, in that case:

minetestmapper supports database backends similar to minetest backends
[example backend](https://github.com/minetest/minetestmapper/blob/master/db-redis.cpp)